### PR TITLE
fix(vector): Bump image to 0.21.1

### PR DESCRIPTION
**Pull request by spencergilbert**
- feat(vector): Bump image to 0.21.1
- chore: changelog

### CODE REVIEW

**Review by jszwedko — COMMENTED** [review]
Thanks! Since the vector version bump is just a patch version bump, I could see doing the same here and have this be `0.10.1` since there should be no functional changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.10.0] - 2022-04-22
```
[the remark this answers is not in the document]
Comment (spencergilbert) [reply CHANGELOG.md:6]: Doh yeah, I done goofed

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.10.1] - 2022-04-22
+
+### Vector
+
+#### Features
```
Comment (jszwedko) [inline CHANGELOG.md:10]: Shouldn't this be "Bug Fixes" 🤔

Comment (jszwedko) [reply CHANGELOG.md]: I rebased and force pushed to rename commit to `fix(vector)`.

**Review by jszwedko — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (spencergilbert) [reply CHANGELOG.md]: Yeah, you right 😵‍💫

**Pull request merged**